apidef/api_definitions.go

```diff
@@ -407,6 +407,7 @@ type VersionDefinition struct {
 	StripPath           bool              `bson:"strip_path" json:"strip_path"` // Deprecated. Use StripVersioningData instead.
 	StripVersioningData bool              `bson:"strip_versioning_data" json:"strip_versioning_data"`
 	Versions            map[string]string `bson:"versions" json:"versions"`
+	BaseID              string            `bson:"base_id" json:"-"`
```

Comment:
bson doesn't match json (needs to be `-`, or json needs an update)?

Reply:
It was by purpose of that we don't want to show base api id to user in API definition.

Reply:
Add the intention in a comment please to make it clear it's intent 👍